tests/drivers/spi/spi_loopback/boards/nrf54h20dk_nrf54h20_cpuapp.overlay

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+&pinctrl {
+	/omit-if-no-ref/ spi130_default: spi130_default {
+		group1 {
+			psels = <NRF_PSEL(SPIM_SCK, 0, 0)>,
+				<NRF_PSEL(SPIM_MISO, 0, 6)>,
+				<NRF_PSEL(SPIM_MOSI, 0, 7)>;
+		};
+	};
+
+	/omit-if-no-ref/ spi130_sleep: spi130_sleep {
```

Comment:
omit-if-no-ref is redundant here, they're always used. also: is it really needed? doesn't board have a set of preferred pins for SPI?

Reply:
No, the board definition doesn't have any preferred SPI pins, I guess they are waiting for a standard header like arduino to be developed.

> omit-if-no-ref is redundant here, they're always used.

Sure will remove

Reply:
A similar approach has been taken from nRF54HL15 as well.

Reply:
There's another PR that got merged with the same overlay, so, dropped that commit.